docker/build-vespa.sh

```diff
@@ -11,8 +11,7 @@ DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
 cd $DIR
 
 VESPA_VERSION=$1
-DOCKER_IMAGE="vespabuild"
+DOCKER_IMAGE="vespaengine/vespa-dev:latest"
```

Comment:
Suggest doing a docker pull vespaengine/vespa-dev to make sure you have the latest. If already have the image with this tag, docker will not automatically pull it.